massdriver/resource_package_alarm.go

```diff
@@ -34,6 +42,39 @@ func resourcePackageAlarm() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"metric": {
```

Comment:
Need guidance here on `Required` and `ForceNew`.

Reply:
Both namespace and name should be required.

statistic is a concept in AWS and Azure but not in GCP, so it shouldn't be required right now.